be/src/storage/rowset/segment_iterator.cpp

```diff
@@ -1409,6 +1409,39 @@ Status SegmentIterator::_do_get_next(Chunk* result, vector<rowid_t>* rowid) {
     MonotonicStopWatch sw;
     sw.start();
 
+    // only used for CACHE SELECT, do not form any chunk to save CPU time,
+    // just read file content in `_scan_range`
+    if (_opts.lake_io_opts.cache_file_only) {
+        // read every column in this segment at once, maybe optimize this later
+        size_t buf_size = config::starlet_fs_stream_buffer_size_bytes;
```

Comment:
`config::starlet_fs_stream_buffer_size_bytes` should be used with `USE_STAROS`

Reply:
done